### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by yahiko615**

### CODE REVIEW

File: task1.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+eleks_employees = ['Misha', 'Dima', 'Marina']
+toshiba_employees = ['Alex', 'Den', 'Marina']
+
+toshiba_employees.extend(eleks_employees)
+print(toshiba_employees)
```
Comment (bohdandovbsyh) [inline task1.py:5]: Not fully completed. Eleks still have employees after your manipulation, but it should not.

File: task3.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+omnivores = {'Bob', 'Charlie'}
+vegetarians = {'Alice', 'Eve', 'Frank'}
+# Didn't quite understand the assignment,
+# it turns out that omnivores can eat both and eat vegetables and herbs,
+# and vegetarians eat vegetables and herbs,
+# so I understand correctly that I just need to combine these two groups?
```
Comment (bohdandovbsyh) [inline task3.py:6]: yes. You are right

**Review by bohdandovbsyh — APPROVED** [review]

File: task1.py (hunk 1 above)
Comment (yahiko615) [reply task1.py:5]: should i clear elex_employees list after extend toshiba?

Comment (yahiko615) [reply task1.py:5]: @bohdandovbsyh

**Pull request merged**